entry_types/scrolled/package/src/frontend/AudioPlayer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {useFile} from '../entryState';
+import {MediaPlayer} from './MediaPlayer';
+
+
+/**
+ * Render audio file in MediaPlayer.
+ *
+ * @param {Object} props
+ * @param {number} props.id - Perma id of the audio file.
+ */
```

Comment:
It's a bit cumbersome, but things with inline docs need to be added to the TOC in `package/documentation.json`. Otherwise the show up at the very end of the sidebar in the [generated docs](https://codevise.github.io/pageflow-docs/scrolled/js/master/index.html).

Reply:
is it `package/documentation.yml`?
I added AudioPlayer and VideoPlayer in there

Reply:
Ah yes, my bad.